README.md

```diff
@@ -260,6 +260,14 @@ Some examples:
    This implies that `/Build/Products/MyMod.framework` exists and contains
    a `.swiftmodule`.  Again the `--source-directory` is searched by default
    if `-F` is not passed in.
+5. With pre-generated symbolgraph files:
+    ```shell
+    jazzy --module MyMod --swift-build-tool symbolgraph
+          --symbolgraph_directory Build/symbolgraphs
```

Comment:
Should be `--symbolgraph-directory` here?  With dash not underscore?

Reply:
Yeah sorry, I'd been staring at the yaml files too much!